x/auth/src/lib.rs

```diff
@@ -11,3 +11,16 @@ pub use genesis::*;
 pub use keeper::*;
 pub use message::*;
 pub use params::*;
+
+//
+
+pub fn new_module_addr(module_name: &str) -> gears::types::address::AccAddress {
```

Comment:
You may use [OnceLock](https://doc.rust-lang.org/std/sync/struct.OnceLock.html)

Reply:
I've updated realization but I cannot use OnceLock because I need to have a mutable instance for creating a hash. The new solution is a little better because I used a special method.